plugins/in_syslog/syslog_prot.c

```diff
@@ -31,23 +31,155 @@ static inline void consume_bytes(char *buf, int bytes, int length)
     memmove(buf, buf + bytes, length - bytes);
 }
 
+static int append_raw_message_to_record_data(char **result_buffer,
```

Comment:
I think you can replace this function with https://github.com/fluent/fluent-bit/blob/master/src/flb_pack.c#L1165